.github/workflows/RELEASE.yaml

```diff
@@ -104,7 +104,7 @@ jobs:
           RELEASE_VERSION: ${{ github.event.release.tag_name }}
 
       - name: Deploy artifacts to Artifactory and Maven Central
-        run: mvn -B compile generate-sources source:jar javadoc:jar deploy -PcheckFormat -Psonatype-oss-release
+        run: mvn -B compile generate-sources source:jar javadoc:jar deploy -PcheckFormat -Psonatype-oss-release -DskipTests
```

Comment:
Do we skip tests because at this point we already ran them in a previous workflow? Just wanna understand.

Reply:
Yes, because main branch is always tested, as well as the feature branches before merging.